portfolio.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html lang="en-GB">
+
+	<head>
+		<title>My Portfolio</title>
+		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
+	</head>	
+	
+	<body>
+		
+		<nav class="nav_bar">
+				<p>Karen Schulz</p>
+			
+			<div>
+				<a href="portfolio.html">Portfolio</a>
+				<a href="contact.html">Contact</a>
+			</div>
+			
+		</nav>
+		
+	
+		<div class="portfolio_page">
+			
+			<img class="mypicture" src="images/profile_pic.png" alt="my_profile_picture">
+		</div>
+			
+		<article class="content">
+			
+			<h1>Welcome to my Portfolio</h1>
+			
+			<div class="portfolio1">
+				<p>
+					<img class="first_work" src="images/green_back.png" alt="my_first_work">
+				</p>
+				
+				<p>I love to do illustrations, check my creations</p>
+				
+			</div>
+			
+		<div class="my_potfolio">
+				
+			<div class="portfolio2">
+				<p>
+					<img class="second_work" src="images/eating.png" alt="my_second_work">
+				</p>
+				<p>I love html and CSS</p>
+			</div>
+				
+			<div class="portfolio3">
+				<p>
+					<img class="third_work" src="images/full-pouch.png" alt="my_third_work">
```

Comment:
for the alt info here and above please give more literal description of what the picture shows =]

Reply:
I added a longer description to the alt